pkg/operator/k8sutil/pod.go

```diff
@@ -76,6 +79,12 @@ func PodIPEnvVar(property string) v1.EnvVar {
 	return v1.EnvVar{Name: property, ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "status.podIP"}}}
 }
 
+// DiscoverDaemonUdevBlacklistEnvVar discover daemon udev blacklist env var
+func DiscoverDaemonUdevBlacklistEnvVar() v1.EnvVar {
+	udevBlacklist := os.Getenv(DiscoverDaemonUdevBlacklistEnvName)
+	return v1.EnvVar{Name: DiscoverDaemonUdevBlacklistEnvName, Value: udevBlacklist}
```

Comment:
```suggestion
	return v1.EnvVar{Name: DiscoverDaemonUdevBlacklistEnvName, Value: os.Getenv(DiscoverDaemonUdevBlacklistEnvName)}
```

Reply:
Since  `udevBlacklist` is used once we don't need it, we can just call `os.Getenv()` directly.